NEWS.md

```diff
@@ -1,5 +1,8 @@
 # pmtables (development version)
 
+- Fix bug where arguments could not be passed along to stable_long 
```

Comment:
do you need to add the font size update here too?

Reply:
@barrettk is reviewing that PR for me; will get that into the news too.

Reply:
I added this but didn't render the document; would cause a conflict with the PR that Kyle is reviewing.